src/App.js

```diff
@@ -23,8 +23,8 @@ class App extends React.Component {
     this.logData = props.logData;
     this.state = {
       timeRange: {
-        minDate: 0,
-        maxDate: new Date("1/1/2050").getTime(),
+        minTime: 0,
+        maxTime: new Date("1/1/2050").getTime(),
```

Comment:
Would it be better to make this a default time window? i.e. `today + 1 year`

Reply:
good idea. will do